db/migrate/20180209180035_add_compliance_information_to_physical_servers.rb

```diff
@@ -0,0 +1,6 @@
+class AddComplianceInformationToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :compliance_name, :string
```

Comment:
Can you change this name so that it doesn't conflict with the builtin MIQ compliances model

Reply:
@agrare I understand, done.